### DISCUSSION TITLE
Is a release for Godot 4 on the horizon?

### DISCUSSION THREAD

**Discussion by jonspe** [abarichello/godot-ci#111] [General]
I saw that there are a couple of commits about adding 4.0 support, but the GitHub Actions Marketplace says the latest release is for Godot 3.4.2. Or is it possible to use the 3.4.2 release with Godot 4? I'm just a little bit confused, since apparently the workflow file is updated for 4.0 but the marketplace version isn't.

**Comment by Calinou** [comment]
- This is already being tracked in https://github.com/abarichello/godot-ci/issues/98.
